### DISCUSSION TITLE
[Feature] Is possible create a simple plugin to execute a script?

### DISCUSSION THREAD

**Discussion by RobertusIT** [FreshRSS/FreshRSS#6690] [Q&A]
I have tried with chatgpt without success. I'm not a dev, but i wish to run a script, when there is new articles in a "Movie" category of FreshRSS.

**Comment by Alkarex** [comment]
You can write an extension, or use an existing one
https://github.com/FreshRSS/Extensions
https://freshrss.github.io/FreshRSS/en/developers/03_Backend/05_Extensions.html

**Reply by RobertusIT** [reply]
> You can write an extension, or use an existing one https://github.com/FreshRSS/Extensions https://freshrss.github.io/FreshRSS/en/developers/03_Backend/05_Extensions.html

Thanks, I already tried to write a plugin but I'm not a dev and with charger no success.

Unfortunately not exist already a plugin that do this simple thing.

I spent 2 days without success. For that I'm here
